mitosheet/mitosheet/tests/test_ts_types.py

```diff
@@ -194,6 +195,13 @@ def test_graph_safety_filter_cutoff_matches():
 
     assert int(graph_safety_filter_cutoff) == GRAPH_SAFETY_FILTER_CUTOFF
 
+def test_user_profile_defaults_matches():
+    user_profile_support_email = get_constant_from_ts_file(
```

Comment:
Can we make the keys of the Mito enterprise config a `MitoEnterpriseConfigKey` enum in the `types.ts` file, and update the tests accordingly? 

I want a test that will fail when we add something in on the frontend or something on the backend without updating the either!